.github/workflows/pythonapp.yml

```diff
@@ -82,9 +82,12 @@ jobs:
         # when there are test failures
         if: always()
 
-      - name: Runs demos
+      - name: Run FFCx and UFL demos
         run: |
-          python -m pytest demo/test_demos.py
+          git clone --branch michal/py-form-files https://github.com/FEniCS/ufl.git
```

Comment:
There are 'actions' form cloning other repos, see https://github.com/FEniCS/basix/blob/main/.github/workflows/dolfin-tests.yml#L46 for an example.